base/cadvisor/cadvisor.ClusterRole.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+    app: cadvisor
+    category: rbac
+    deploy: sourcegraph
+    sourcegraph-resource-requires: no-cluster-admin
```

Comment:
this requires cluster-admin

Reply:
```suggestion
    sourcegraph-resource-requires: cluster-admin
```

Reply:
oh hm I saw that prometheus had some conditions in the no-privilege overlay so I used the same labels and whatnot, thanks!